test/jdk/java/lang/StringBuilder/StringBufferRepeat.java

```diff
@@ -129,6 +129,19 @@ public void sanity() {
         expected = "\u0000\u0000\u0000\u0000\u0000\u0000\u0020\u0020\u0020\u0020\u0020\u0020\u2461\u2462\u2462\u2462\u2462\u2462\udbff\udfff\udbff\udfff\udbff\udfff\udbff\udfff\udbff\udfff\udbff\udfff";
         assertEquals(expected, sb.toString());
 
+        // toStringCache
+
+        sb.setLength(0);
+        sb.toString();
+        sb.repeat('*', 5);
+        expected = "*****";
+        assertEquals(expected, sb.toString());
```

Comment:
Hello Jim, just a minor detail - in case of testng's `Assert.assertEquals()` the first param is the `actual` and the second is the `expected`. Some other existing usages of this method in this test also have the incorrect order.

Reply:
Updated